wallet/src/de/schildbach/wallet/ui/MoreFragment.kt

```diff
@@ -18,80 +18,59 @@ package de.schildbach.wallet.ui
 
 import android.content.Intent
 import android.os.Bundle
-import android.view.MenuItem
-import androidx.appcompat.widget.Toolbar
+import android.view.View
+import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import de.schildbach.wallet.AppDatabase
 import de.schildbach.wallet.WalletApplication
 import de.schildbach.wallet.data.BlockchainState
-import de.schildbach.wallet.util.showBlockchainSyncingMessage
 import de.schildbach.wallet_test.R
 import kotlinx.android.synthetic.main.activity_more.*
 import org.dash.wallet.integration.uphold.ui.UpholdAccountActivity
 
-class MoreActivity : GlobalFooterActivity() {
+class MoreFragment : Fragment(R.layout.activity_more) {
 
     private var blockchainState: BlockchainState? = null
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContentViewWithFooter(R.layout.activity_more)
-        activateMoreButton()
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
 
-        val toolbar = findViewById<Toolbar>(R.id.toolbar)
-        setSupportActionBar(toolbar)
-        val actionBar = supportActionBar
-        actionBar?.apply {
-            setDisplayHomeAsUpEnabled(true)
-            setDisplayShowHomeEnabled(true)
-        }
-
-        setTitle(R.string.more_title)
-
-        AppDatabase.getAppDatabase().blockchainStateDao().load().observe(this, Observer {
+        setupActionBarWithTitle(R.string.more_title)
+        AppDatabase.getAppDatabase().blockchainStateDao().load().observe(viewLifecycleOwner, Observer {
             blockchainState = it
         })
 
         buy_and_sell.setOnClickListener {
             if (blockchainState != null && blockchainState?.replaying!!) {
-                showBlockchainSyncingMessage()
+                //TODO: ???
+                //showBlockchainSyncingMessage()
```

Comment:
Do you have a question on why this is here?  On why we can't access Uphold if the blockchain is syncing?

Reply:
I wasn't finding this method. It's on `ActivityExtensions`. Fixed on https://github.com/dashevo/dash-wallet/pull/462/commits/ce35cdaccf4394b27b37dd901302c3337c1b37ec.